src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<ul><li>"Contrary" --&gt; "Conversely"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Actually, "Conversely to ..." is apparently rarely (even not?) used. Conversely (:P), "Contrary to..." looks very common.  </p>

Reply:
Left it at Contrary, it okay ?